internal/cmd/config.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
 
+	"github.com/banzaicloud/pipeline/pkg/values"
```

Comment:
This should go after [`"github.com/banzaicloud/pipeline/pkg/cluster"`](https://github.com/banzaicloud/pipeline/pull/2757/files#diff-d8a090a7190ca32eda29d0cd6fa8703aR40)